quiche/src/h3/mod.rs

```diff
@@ -1280,6 +1296,17 @@ impl Connection {
             conn.dgram_max_writable_len().is_some()
     }
 
+    /// Returns whether the peer enabled HTTP/3 CONNECT protocol support.
+    ///
+    /// Support is signalled by the peer's SETTINGS, so this method always
+    /// returns false until they have been processed using the [`poll()`]
+    /// method.
+    ///
+    /// [`poll()`]: struct.Connection.html#method.poll
+    pub fn connect_protocol_enabled_by_peer(&self) -> bool {
```

Comment:
for consistency with `enable_extended_connect`, how about 
```suggestion
    pub fn extended_connect_enabled_by_peer(&self) -> bool {
```